src/classes/UTIL_RecordTypes_TEST.cls

```diff
@@ -99,4 +100,59 @@ private class UTIL_RecordTypes_TEST {
         }
     }
 
+    //Helper Methods
+        /**
+    * @description Return a random valid record type for the sObject
+    * @param objectType The sObject type of the record type
+    * @return Id
+    */
+    public static Id getRandomRecordTypeId(sObjectType objectType) {
+        List<Schema.RecordTypeInfo> recordTypeInfos = objectType.getDescribe().getRecordTypeInfos();
+        for (Schema.RecordTypeInfo rti : recordTypeInfos) {
+            if (rti.isAvailable()) {
+                return rti.getRecordTypeId();
+            }
+        }
+        return null;
+    }
+
+    /**
+    * @description Return the id of the active non-default record type of the sObject for the current user
+    * @param objectType The sObject type of the record type
+    * @return Id
+    */
+    public static Id getNonDefaultRecordTypeId(sObjectType objectType) {
+        return getRecordType(objectType, false);
+    }
+
+    /**
+    * @description Return the id of the active default record type of the sObject for the current user
+    * @param objectType The sObject type of the record type
+    * @return Id
+    */
+    public static Id getDefaultRecordTypeId(sObjectType objectType) {
+        return getRecordType(objectType, true);
+    }
+
+    /**
+    * @description Return the id of an active default/non default record type of the sObject for the current user
+    * @param objectType The sObject type of the record type
+    * @param isDefault Boolean to determine if the default record type is return or not
+    * @return Id
+    */
+    private static Id getRecordType(sObjectType objectType, Boolean isDefault) {
```

Comment:
Let's rename method to `getRecordTypeId`.
Let's use `SObjectType` per Apex code style guidelines (there are more instances in this file).